### PR TITLE
OS X Application UX changes.

### LINKED ISSUE #276: OS X Application UX changes.
**Branch:** `feature/osx-o.16.x` (osx installer)

- [x] remove the 'Setup' button
- [x] remove the spinner icon on the logo
- [x] remove the 'Advanced' tab
- [x] use 'kalite --version' to make the ka-lite version dynam

### PR DESCRIPTION
**Pull request by amodia**
@cpauya this fixes #276 

Here's the sample screenshot of the preferences menu.

screen shot 2015-11-18 at 5 09 56 pm

### CODE REVIEW

File: osx/KA-Lite/KA-Lite/AppDelegate.m [hunk 1]
```diff
@@ -78,9 +78,12 @@ - (void)applicationDidFinishLaunching:(NSNotification *)aNotification {
             showNotification(@"Kalite executable not found.");
             
         } else {
+            
+            [self runKalite:@"--version"];
             NSLog([NSString stringWithFormat:@"FOUND kalite at %@!", kalite]);
             showNotification(@"KA Lite is now loaded.");
             [self getKaliteStatus];
+
```
Comment (cpauya) [inline osx/KA-Lite/KA-Lite/AppDelegate.m]: @amodia Let's remove the extra blank lines and can you move the "--version" line before the "getKaliteStatus" line so that we log first then execute the command next.

File: osx/KA-Lite/KA-Lite/AppDelegate.m [hunk 2]
```diff
@@ -506,7 +511,6 @@ - (void)showStatus:(enum kaliteStatus)status {
             self.stopButton.enabled = NO;
             self.openBrowserButton.enabled = NO;
             [self.statusItem setToolTip:@"KA-Lite is starting..."];
-            [self.statusItem setImage:[NSImage imageNamed:@"loading"]];
```
Comment (cpauya) [inline osx/KA-Lite/KA-Lite/AppDelegate.m]: Why did you remove the "loading" icon when status is starting-up?

**Comment by cpauya** [comment]
@amodia Pls remove the spinner image on the logo.

File: osx/KA-Lite/KA-Lite/AppDelegate.m (hunk 2 above)
Comment (amodia) [inline osx/KA-Lite/KA-Lite/AppDelegate.m]: @cpauya  That line is to remove the spinner image on the logo.

Comment (cpauya) [inline osx/KA-Lite/KA-Lite/AppDelegate.m]: Nope, that line is at https://github.com/mrpau/installers/blob/feature/276-os-x-application-ux-changes/osx/KA-Lite/KA-Lite/AppDelegate.m#L512 with this code:

> [self.statusItem setImage:[NSImage imageNamed:@"loading"]];

**Comment by cpauya** [comment]
@amodia Any screenshot of how it looks like now?

**Comment by amodia** [comment]
@cpauya here's the current screenshot of the preferences menu.

screen shot 2015-11-18 at 9 08 56 pm

**Comment by cpauya** [comment]
Good one, merging.

**Comment by jamalex** [comment]
Yayyyyyyyy!
